src/univers/version_range.py

```diff
@@ -319,6 +351,37 @@ def from_native(cls, string):
 
         return cls(constraints=constraints)
 
+    @classmethod
+    def from_gitlab_native(cls, string):
+        constraints = []
+        parsed_constraints = []
+        pipe_separated_constraints = string.split("||")
+        for pipe_separated_constraints in pipe_separated_constraints:
+            space_seperated_constraints = pipe_separated_constraints.split(" ")
+            constraints.extend(space_seperated_constraints)
+        comparator = ""
+        while "" in constraints:
```

Comment:
May be just skip empty constraints in the loop below instead? the while/remove is a somewhat uncommon